src/app/work-item/work-item.component.html

```diff
@@ -29,8 +29,29 @@ <h2 class="card-pf-title text-center">
     <div class="row toolbar-pf">
       <div class="col-md-12 col-sm-12 col-xs-12">
         <div class="list-action toolbar-pf-actions">
+          <!--Filter-->
+          <div class="form-group toolbar-pf-filter">
+            <label class="sr-only" for="filterwitype">Filter</label>
+            <div class="input-group">
+              <div class="input-group-btn" dropdown>
+                <button class="btn btn-default dropdown-toggle" type="button" id="wi_filter_dropdown" data-toggle="dropdown" dropdown-open>
+                  Filter
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu" aria-labelledby="wi_filter_dropdown">
+                  <li *ngIf="!filters.length">
+                    <span>No filters available</span>
+                  </li>
+                  <li *ngFor="let filter of filters;" role="presentation" (click)="activeFilter(filter.id)">
```

Comment:
@SMahil @nimishamukherjee The Filter Query Dropdown Filter Bar should be utilized:
![screen shot 2017-02-07 at 8 33 35 am](https://cloud.githubusercontent.com/assets/4032718/22693199/3d271686-ed10-11e6-9601-37f2682bf522.png)
http://www.patternfly.org/pattern-library/forms-and-controls/filter/#/_design

Accordingly, it should be as such:
![screen shot 2017-02-07 at 8 44 54 am](https://cloud.githubusercontent.com/assets/4032718/22693553/bdee92de-ed11-11e6-991f-682044f3c943.png)